{"name": "Login table Examples invalid to OrangeHRM -- @1.2 ", "status": "failed", "statusDetails": {"message": "AssertionError\n", "trace": "  File \"/usr/local/lib/python3.8/site-packages/behave/model.py\", line 1806, in run\n    match.run(runner.context)\n  File \"/usr/local/lib/python3.8/site-packages/behave/matchers.py\", line 97, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/orangeHrm_steps.py\", line 57, in login_error\n    assert result is False\n"}, "steps": [{"name": "Given the user is on the search page \"orangehrmlive\"", "status": "passed", "start": 1608998294490, "stop": 1608998297873}, {"name": "When insert username \"admin1\" and password \"admin123\"", "status": "passed", "start": 1608998297874, "stop": 1608998298042}, {"name": "And Click on login button", "status": "passed", "start": 1608998298042, "stop": 1608998298662}, {"name": "Then User must unsuccesfully login to the Dashboard page", "status": "failed", "statusDetails": {"message": "AssertionError\n", "trace": "  File \"/usr/local/lib/python3.8/site-packages/behave/model.py\", line 1806, in run\n    match.run(runner.context)\n  File \"/usr/local/lib/python3.8/site-packages/behave/matchers.py\", line 97, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/orangeHrm_steps.py\", line 57, in login_error\n    assert result is False\n"}, "start": 1608998298663, "stop": 1608998298692}], "parameters": [{"name": "name", "value": "admin1"}, {"name": "pwd", "value": "admin123"}], "start": 1608998292250, "stop": 1608998298803, "uuid": "03389e0f-45bc-4f38-ae23-a8d7d7a38ab4", "historyId": "2bad80baf8441bdca773af0bc48122ea", "fullName": "features/orangeHrm/orangeHrm_outline.feature:Login table Examples invalid to OrangeHRM", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "test"}, {"name": "feature", "value": "OrangeHRM outline"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}